### ISSUE TITLE
Questions about base image

### ISSUE THREAD

**Issue by JokerQyou** [ishani/xSyn#3]
First of all thank you for your effort to provide such a lean and compact implementation of xSyncBrowser api server.
I noticed the base image you used was `centurylink/ca-certs`. According to the DockerHub page of that image, it was meant to be used as a base image for images that `require certificate verification for out-bound SSL connections`. xSync is a server if I read it correctly, so what functionality is it to require out-bound SSL connections? Is it the ACME-related certificate automation?
Another question is, the `centurylink/ca-certs` image you used has not been updated for a long time, and its CA cert list might probably be outdated. Shall we consider using another actively maintained `ca-certs` image instead?

**Comment by ishani** [comment]
You're quite right! That image has rotted over time, it's just my muscle memory for building Go containers.

I'm going to switch it to a multi-stage process that also gets-and-builds the server inside a golang container, ie: https://bl.ocks.org/nicerobot/c11ee7e2e2b3135e8df8083a585607e2

.. which is much neater and should have a recent cert set
